graspy/simulations/__init__.py

```diff
@@ -1,3 +1,16 @@
-from .simulations import *
-from .simulations_corr import *
-from .rdpg_corr import *
+from .simulations import sample_edges, er_np, er_nm, sbm, rdpg, p_from_latent
+from .simulations_corr import sample_edges_corr, er_corr, sbm_corr
+from .rdpg_corr import rdpg_corr
+
+__all__ = [
+    "sample_edges",
+    "er_np",
+    "er_nm",
+    "sbm",
+    "rdpg",
+    "p_from_latent",
+    "sample_edges_corr",
+    "er_corr",
+    "sbm_corr",
+    "rdpg_corr",
```

Comment:
```suggestion
    "rdpg_corr"
```
Think this is whats causing your errors. Remove the trailing comma and you should be good

Reply:
Black doesn't let me get rid of the trailing comma, also the __init__.py for embed also has that trailing comma

Reply:
@asaadeldin11 trailing commas in lists shouldn't cause any issues, see [pep-8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas) on it for example